.golangci.yml

```diff
@@ -3,6 +3,7 @@ linters:
   disable:
     - megacheck
     - stylecheck
+    - funlen
```

Comment:
This is a "new" linter since we bumped the version of `golangci-lint` in #121. While I tend to prefer small functions, we should not be pleasing linters and discussing how many lines is small or not.
In particular, this linter complains about perfectly fine table test functions with a few test cases.